util/Radiance_Monitor/nwprod/gdas_radmon/fix/gdas_radmon_scaninfo.txt

```diff
@@ -54,10 +54,15 @@
  ssu                    -35.000    10.000       90
  hsb                    -48.950     1.100       90
  seviri_m08               0.00      1.00        90
+ seviri_m09               0.00      1.00        90
  seviri_m10               0.00      1.00        90
  seviri_m11               0.00      1.00        90
  saphir_meghat          -42.960     0.666      130
  abi_g16                  0.00      1.00        90
- ahi_himawari8            0.00      1.00        90
  abi_g17                  0.00      1.00        90
+ abi_g18                  0.00      1.00        90
+ ahi_himawari8            0.00      1.00        90
+ ahi_himawari9            0.00      1.00        90
+ atms_n21               -52.393     1.10        96
+ cris-fsr_n21           -48.130     3.333       30 
```

Comment:
Question regarding addition of viirs already mentioned in this PR.

Reply:
> When we update `global_convinfo.txt`, `global_ozninfo.txt`, `global_satinfo.txt`, or `prepobs_errtable.global`, we usually update files in or add files to `fix/gfsv16_historical`. Retrospective parallels often use fix files from `fix/gfsv16_historical`. This PR adds new data in monitor mode. Thus, it may not be necessary for this PR to update or add files to `fix/gfsv16_historical`. It's important to review this point prior to starting GFS v16.x retrospective parallels.
> 
> We have multiple GFS V16.x PRs altering fix files. For example, this PR, #345, and PR #343 both alter fix files. PR #343 was merged into the master today (3/23/2022). PR #343 updated `global_convinfo.txt` and `prepobs_errtable.global`. These changes do not conflict with this PR. Still care should be taken to ensure the final fix in NOAA-EMC/GSI master contains all GFS v16.x fix changes. It's easy for a desired / required change to be missed or undone.

I will carefully examine the fix directory to make sure this PR have the changes from PR #343

Reply:
Thanks!